### ISSUE TITLE
aria-label warning causes test failure

### ISSUE THREAD

**Issue by nicksellen** [yunity/foodsaving-frontend#406]
Maybe just because I have super new chromium my tests fail with:

```
FAILED TESTS:
  StoreList
    ✖ "after each" hook for "gets store data"
      Chrome 56.0.2924 (Linux 0.0.0)
    Error: Expected $log to be empty! Either a message was logged unexpectedly, or an expected log message was not checked and removed:
    ---------
    MOCK $log (warn): ARIA: Attribute "
    
    ---------
    MOCK $log (warn): aria-label
    
    ---------
    MOCK $log (warn): ", required for accessibility, is missing on node:
    
    ---------
    MOCK $log (warn): [object HTMLButtonElement]
    
    ---------
    
        at Object.$log.assertEmpty (webpack:///~/angular-mocks/angular-mocks.js:420:0 <- spec.bundle.js:33410:16)
        at Context.<anonymous> (webpack:///client/app/components/group/_storeList/storeList.spec.js:13:9 <- spec.bundle.js:140638:11)
```

... fix incoming shortly ...
